### ISSUE TITLE
Scene control category buttons do not have tooltips

### ISSUE THREAD

**Issue by Fyorl** [foundryvtt/foundryvtt#9948]
Originally reported by atinvexien https://discord.com/channels/170995199584108546/1065764070680186930/1143216167725584607

**ALL MODULES DISABLED?** True
**OS, Hosting, Browser (if applicable)**: Windows 10, Self-hosting, Foundry application (11.307)
**Short Description of bug**: There is no hover text on the left-side menu buttons (only on the top-level buttons, the secondary menu of buttons with the new hover text works)
**Simple steps to reproduce the bug**: Hover over the left-side menu buttons. Observe the lack of hover text.
**Screenshots and/or console errors**: N/A
